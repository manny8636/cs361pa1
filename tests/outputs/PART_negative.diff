<<<<<<< HEAD
--- outputs/PART_negative.txt	2022-02-14 17:13:29.640858005 -0500
+++ expected/PART_negative.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
-Parsing ../data/negative.txt failed
=======
--- outputs/PART_negative.txt	2022-02-14 17:19:50.520428242 -0500
+++ expected/PART_negative.txt	2022-02-13 17:48:24.374294593 -0500
@@ -0,0 +1,2 @@
>>>>>>> 149606adc8b3f0cb79ed9f7c590375c37f62be46
+INTEGER: -25
+Success!
