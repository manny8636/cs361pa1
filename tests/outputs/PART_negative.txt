Parsing ../data/negative.txt failed
