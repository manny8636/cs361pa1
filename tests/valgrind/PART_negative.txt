==372074== Memcheck, a memory error detector
==372074== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372074== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372074== Command: ../json -i ../data/negative.txt
==372074== 
==372074== 
==372074== HEAP SUMMARY:
==372074==     in use at exit: 2,064 bytes in 1 blocks
==372074==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372074== 
==372074== LEAK SUMMARY:
==372074==    definitely lost: 2,064 bytes in 1 blocks
==372074==    indirectly lost: 0 bytes in 0 blocks
==372074==      possibly lost: 0 bytes in 0 blocks
==372074==    still reachable: 0 bytes in 0 blocks
==372074==         suppressed: 0 bytes in 0 blocks
==372074== Rerun with --leak-check=full to see details of leaked memory
==372074== 
==372074== For lists of detected and suppressed errors, rerun with: -s
==372074== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
