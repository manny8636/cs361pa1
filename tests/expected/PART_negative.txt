INTEGER: -25
Success!
